packages/ra-ui-materialui/src/layout/TitlePortal.stories.tsx

```diff
@@ -28,3 +32,21 @@ export const Sx = () => (
         <Title title="Hello, world" />
     </TestMemoryRouter>
 );
+
+export const NonConfigurable = ({ preferenceKey = 'title' }) => (
+    <TestMemoryRouter>
+        <PreferencesEditorContextProvider>
+            <Inspector />
+            <InspectorButton />
+            <Configurable
```

Comment:
**suggestion**: This story is misleading because we see that we can edit the title but we cannot save it. I think there is no need to wrap `<TitlePortal>` and `<Title>` in a `<Configurable>` component, since `<Title>` already does that.